pkgs/by-name/sp/speakersafetyd/package.nix

```
@@ -2,23 +2,29 @@
   stdenv,
   lib,
   rustPlatform,
-  fetchCrate,
+  fetchFromGitHub,
   pkg-config,
   alsa-lib,
-  rust,
   udevCheckHook,
+  nix-update-script,
 }:
 
 rustPlatform.buildRustPackage rec {
```

Maybe also
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```
? It's now supported and preferred over rec

---

Missed that, applied.